cpp/src/gandiva/date_utils.h

```diff
@@ -47,6 +57,38 @@ class DateUtils {
   static std::vector<std::string> GetExactMatches(const std::string& pattern);
 };
 
+namespace internal {
+
+/// \brief Returns seconds since the UNIX epoch
+static inline bool ParseTimestamp(const char* buf, const char* format, int64_t* out) {
+#if defined(_MSC_VER)
+  static std::locale lc_all(setlocale(LC_ALL, NULLPTR));
+  std::istringstream stream(buf);
```

Comment:
on linux, we've had issues with using streams in IR code, especially with static libs.

but, we can fix this in a subsequent PR, if required.

Reply:
please ignore my comment  - this file isn't used in the IR code.